### ISSUE TITLE
PHP v5.5.3: using utf8mb4 instead of utf8 

### ISSUE THREAD

**Issue by jeroendesloovere** [forkcms/forkcms#1299]
### PHP Version 5.5.3 introduced "utf8mb4", which is recommended
```yml
collation-server     = utf8mb4_general_ci # Replaces utf8_general_ci
character-set-server = utf8mb4            # Replaces utf8
```
We recommend against MySQL's utf8 character set, since it does not support 4-byte unicode characters, and strings containing them will be truncated. This is fixed by the newer utf8mb4 character set.

[Source](http://symfony.com/doc/current/book/doctrine.html)

**Comment by WouterSioen** [comment]
:+1:

**Comment by WouterSioen** [comment]
Will you create a PR for this?

**Comment by WouterSioen** [comment]
poke @jeroendesloovere

**Comment by jeroendesloovere** [comment]
I've got [a utf8mb4 branch](https://github.com/jeroendesloovere/forkcms/tree/utf8mb4-integration) for this.
But installation is breaking and throwing errors.

Still looking into this...

**Comment by WouterSioen** [comment]
nice that there is progress here!

**Comment by GhaziTriki** [comment]
Please take a look at this to understand the difference between utf8mb4_general_ci and utf8mb4_unicode_ci http://stackoverflow.com/questions/766809/whats-the-difference-between-utf8-general-ci-and-utf8-unicode-ci

If Fork CMS is going to support non latin characters then using a utf8mb4_unicode collation is better.

**Comment by jeroendesloovere** [comment]
"There is almost certainly no reason to use utf8mb4_general_ci anymore, as we have left behind the point where CPU speed is low enough that the performance difference would be important. Your database will almost certainly be limited by other bottlenecks than this."
=> We will use utf8mb4_unicode...

**Comment by GhaziTriki** [comment]
:smiley: :+1:

**Comment by WouterSioen** [comment]
+1 for latin!

**Comment by GhaziTriki** [comment]
@jeroendesloovere is it possible to provide a PR with utf8_mb4 changes?

**Comment by jeroendesloovere** [comment]
@GhaziTriki Sure I can do that.
Please beware that it is a non-working PR.

**Comment by GhaziTriki** [comment]
@jeroendesloovere What issues are expected?

**Comment by WouterSioen** [comment]
fixed in 4.0.0
